app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/meldekort/BrukersMeldekortPostgresRepo.kt

```diff
@@ -30,22 +30,28 @@ class BrukersMeldekortPostgresRepo(
                         meldeperiode_versjon,
                         sak_id,
                         mottatt,
-                        dager
+                        dager,
+                        journalpost_id,
+                        oppgave_id
                     ) values (
                         :id,
                         :meldeperiode_hendelse_id,
                         (SELECT id FROM meldeperiode WHERE hendelse_id = :meldeperiode_hendelse_id),
                         (SELECT versjon FROM meldeperiode WHERE hendelse_id = :meldeperiode_hendelse_id),
                         :sak_id,
                         :mottatt,
-                        to_jsonb(:dager::jsonb)
+                        to_jsonb(:dager::jsonb),
+                        :journalpost_id,
+                        :oppgave_id
                     )
                     """,
                     "id" to brukersMeldekort.id.toString(),
                     "meldeperiode_hendelse_id" to brukersMeldekort.meldeperiodeHendelseId.toString(),
                     "sak_id" to brukersMeldekort.sakId.toString(),
                     "mottatt" to brukersMeldekort.mottatt,
                     "dager" to brukersMeldekort.toDagerJson(),
+                    "journalpost_id" to brukersMeldekort.journalpostId.toString(),
+                    "oppgave_id" to brukersMeldekort.oppgaveId,
```

Comment:
Hvis oppgaveId er et objekt må du kanskje ha noe toString eller value eller noe slikt for å hente ut det som skal skrives til databasen så du ikke får en objektreferanse :)